_gsocprojects/2023/project_MadAnalysis5.md

```diff
@@ -0,0 +1,23 @@
+---
+project: MadAnalysis5
+layout: default
+logo: MadAnalysis5-logo.png
+description: |
+    [MadAnalysis 5](http://madanalysis.irmp.ucl.ac.be) is a framework for phenomenological investigations at particle 
+    colliders. Based on a C++ kernel, this program allows to efficiently perform, in a straightforward and user-friendly fashion, sophisticated physics 
+    analyses of event files such as those generated a large class of Monte Carlo event generators. MadAnalysis 5 comes 
```

Comment:
not sure the syntax of that sentence is correct

Reply:
Thanks @hegner, this has been updated.